### DISCUSSION TITLE
so how to install ?

### DISCUSSION THREAD

**Discussion by samozin** [envision-mods/portal#2] [General]
i will try to install it manully... hope to see something great as u are :)

**Comment by live627** [comment]
I see that have a fan. Hi.

Coincidentally, I have recently  gained a newfound interest in this after more than a decade. As such, the code is over ten years old (some parts 15!) and throw errors on new PHP because of the usage of deprecated functions. I'm rewriting it all to work in modern PHP.

So you won't get an awesome working script just yet!

I'm going to move this to discussions where it fits better.
